include/memkind.h

```diff
@@ -146,6 +146,54 @@ enum {
     MEMKIND_ERROR_RUNTIME = -255                /**<  Error: Unspecified run-time error */
 };
 
+/* KIND CONFIGURATION MANAGEMENT INTERFACE */
+
+/// \brief Memkind memory usage policy
+typedef enum memkind_mem_usage_policy {
+    MEMKIND_MEM_USAGE_POLICY_DEFAULT      = 0,        /**<  Default  memory usage  */
+    MEMKIND_MEM_USAGE_POLICY_CONSERVATIVE = 1,        /**<  Minimize memory usage at all costs, */
+    MEMKIND_MEM_USAGE_POLICY_MAX_VALUE
+} memkind_mem_usage_policy;
+
+///
+/// \brief Create a memkind configuration
+/// \note STANDARD API
+/// \return Memkind configuration, NULL on failure
+///
+struct memkind_config *memkind_create_config(void);
```

Comment:
`memkind_config_new`

(create implies creation of the object, not necessarily initialization).